src/czechpmdevs/multiworld/command/subcommand/UnloadSubCommand.php

```diff
@@ -35,6 +35,11 @@ public function execute(CommandSender $sender, array $args, string $name): void
 			$sender->sendMessage(LanguageManager::translateMessage($sender, "unload-usage"));
 			return;
 		}
+		
+		if(!Server::getInstance()->getWorldManager()->getDefaultWorld()->getFolderName() == $args[0]) {
```

Comment:
This codestyle looks weird. Use either `!==` or `===`, style `! ... ==` doesn't match project code style.